app.rb

```diff
@@ -0,0 +1,249 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'data/data_manager'
+require_relative 'data/data_handler'
+
+class App
```

Comment:
Running the command `ruby main.rb`

Error 👀
cannot load such file -- `.../school-library/data/data_manager` (LoadError)

![image](https://github.com/Paulinelali/school-library/assets/78251927/83c8f481-6dbb-48c8-ba9f-da844ce0aca1)

- [ ] I noticed you are trying to load some files e.g. `data/data_manager` but I could not find any file with that name in your app, and this is the (LoadError). 

- [ ] `require_relative` complements the built-in method `require` by allowing you to load a file relative to the file containing the `require_relative` statement, It does not search in the Ruby load path or gems

- [ ] According to the [project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md), it is important to make sure that our console app works as expected. our school library console app should behave like the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) provided for us 😊.

Please fix this 🛠💪

Reply:
But I have it running on my PC, this is some kind of error that may be from mine or your end, I'm not sure. Please can you retry again? I just deleted and added it again now

Reply:
Hi @Paulinelali, you app runs properly without errors